packages/core/lib/env/aws_lambda.js

```diff
@@ -24,14 +24,16 @@ module.exports.init = function init() {
     logger.getLogger().warn('AWS Lambda does not support AWS X-Ray manual mode.');
   };
 
-  fs.mkdir('/tmp/', function() {
-    fs.mkdir('/tmp/.aws-xray/', function() {
-      var filename = '/tmp/.aws-xray/initialized';
-      fs.closeSync(fs.openSync(filename, 'a'));
-      var now = new Date();
-      fs.utimesSync(filename, now, now);
-    });
-  });
+  try {
+    fs.mkdirSync('/tmp/.aws-xray/');
+  } catch (e) {
+    logger.getLogger().debug('Lambda handshake directory already exists.');
```

Comment:
This catches all issues and not just if the directory exists so the log entry isn't accurate?